### DISCUSSION TITLE
getShopByOwnerUserId Actually Requires OAuth

### DISCUSSION THREAD

**Discussion by tophattwaffle** [etsy/open-api#834] [Q&A] [answered]
When hitting the `getShopByOwnerUserId` endpoint using just the API key as specified in the documentation I get `{"error":"No user was provided as part of the bearer token."}`

Once I make the request with my bearer token from OAuth it works fine. Am I doing something wrong?

![image](https://user-images.githubusercontent.com/6774125/214781644-60043084-1dac-40d5-a282-be5d89bbff5b.png)

**Comment by Gareth064** [comment]
Just checking... You are providing a user_id as a path parameter in your request?

![Screenshot_20230126-175840.png](https://user-images.githubusercontent.com/22307223/214913149-77f245df-7c2c-408b-83fd-8ad6d8afcf6e.png)

**Reply by Gareth064** [reply]
Never mind .. I see it in your screenshot.

**Comment by tophattwaffle** [comment]
Yeah, it's very strange. This isn't blocking me from anything since I'm just sending my bearer token with for now. Mostly wanted to confirm that the documentation seems to be incorrect.

**Comment by MickaelBNeron** [comment, marked as answer]
I ran into the exact same issue. The documentation is incorrect: a "Authorization" header with the access token as a Bearer token is actually required.

**Comment by simonj** [comment]
Can we fix this issue 😅
